types/mjml-react/mjml-react-tests.tsx

```diff
@@ -240,22 +260,70 @@ function renderOutTestEmail() {
 {
     {
         const minProps: React.ReactNode = <MjmlAccordion />;
-        const maxProps: React.ReactNode = <MjmlAccordion>child</MjmlAccordion>;
+        const maxProps: React.ReactNode = (
+            <MjmlAccordion
+                border="0"
+                containerBackgroundColor="#ffffff"
+                fontFamily="Arial"
+                iconAlign="left"
+                iconHeight="20px"
+                iconPosition="right"
+                iconUnwrappedAlt="close"
+                iconUnwrappedUrl="open.png"
+                iconWidth="20px"
+                iconWrappedAlt="open"
+                iconWrappedUrl="close.png"
+            >
+                child
+            </MjmlAccordion>
+        );
     }
     // MjmlAccordionElement
     {
         const minProps: React.ReactNode = <MjmlAccordionElement />;
-        const maxProps: React.ReactNode = <MjmlAccordionElement>child</MjmlAccordionElement>;
-    }
-    // MjmlAccordionText
-    {
-        const minProps: React.ReactNode = <MjmlAccordionText />;
-        const maxProps: React.ReactNode = <MjmlAccordionText>child</MjmlAccordionText>;
+        const maxProps: React.ReactNode = (
+            <MjmlAccordionElement
+                border="0"
+                backgroundColor="#ffffff"
+                fontFamily="Arial"
+                iconAlign="left"
+                iconHeight="20px"
+                iconPosition="right"
+                iconUnwrappedAlt="close"
+                iconUnwrappedUrl="open.png"
+                iconWidth="20px"
+                iconWrappedAlt="open"
+                iconWrappedUrl="close.png"
+            >
+                child
+            </MjmlAccordionElement>
+        );
     }
     // TestMjmlAccordionTitleTag
     {
         const minProps: React.ReactNode = <MjmlAccordionTitle />;
-        const maxProps: React.ReactNode = <MjmlAccordionTitle>child</MjmlAccordionTitle>;
+        const maxProps: React.ReactNode = (
+            <MjmlAccordionTitle backgroundColor="#ffffff" color="#000000" fontFamily="Arial" fontSize="20px">
+                child
+            </MjmlAccordionTitle>
+        );
+    }
+    // MjmlAccordionText
+    {
+        const minProps: React.ReactNode = <MjmlAccordionText />;
+        const maxProps: React.ReactNode = (
+            <MjmlAccordionText
+                backgroundColor="#ffffff"
+                color="#000000"
+                fontFamily="Arial"
+                fontSize="20px"
+                fontWeight="5"
+                letterSpacing="1.5"
+                lineHeight="2"
+            >
+                child
+            </MjmlAccordionText>
+        );
```

Comment:
When adding the tests I new-lined maxProps due to amount of props applied when using all. I'll flatten these onto one line if preferred though(?)